source/funkin/ui/debug/charting/ChartEditorState.hx

```diff
@@ -258,16 +259,6 @@ class ChartEditorState extends UIState // UIState derives from MusicBeatState
    */
   public static final BASE_QUANT_INDEX:Int = 3;
 
-  /**
-   * The duration before the welcome music starts to fade back in after the user stops playing music in the chart editor.
-   */
-  public static final WELCOME_MUSIC_FADE_IN_DELAY:Float = 30.0;
-
-  /**
-   * The duration of the welcome music fade in.
-   */
-  public static final WELCOME_MUSIC_FADE_IN_DURATION:Float = 10.0;
```

Comment:
These got converted from constants to variables because they could be modified by the sliders, but didn't get changed back.